drivers/xen/net/Config.uk

```diff
@@ -0,0 +1,7 @@
+config XEN_NETFRONT
```

Comment:
The convention is to prefix with `LIB`, so that would be `LIBXEN_NETFRONT`. Similarly for the other drivers.